.github/workflows/ci.yml

```diff
@@ -109,24 +109,38 @@ jobs:
 
     strategy:
       matrix:
+        runtime: [node, nsolid]
         node-version: [14, 16, 18, 20]
         os: [macos-latest, ubuntu-latest, windows-latest]
+        include:
+          - runtime: nsolid
+            nsolid-version: 5
         exclude:
-          # excludes node 14 on Windows
           - os: windows-latest
+            runtime: node
             node-version: 14
-
+          - runtime: nsolid
+            node-version: 14
```

Comment:
Considering nsolid supports by default only the latest LTS (in this case, 20). I think we don't need to include a 'wontfix' release line here.

Reply:
Ey thanks for taking a look to this PR.
Since we're testing NSOLID 5 which only supports LTS version Node 18 and Node 20, we need to exclude node14 and node16 from the matrix because by default It rollout a Job for `node-version: [14, 16, 18, 20]`